cupy/random/_generator_api.pyx

```diff
@@ -521,7 +521,9 @@ cdef void _launch_dist(bit_generator, func, out, args) except*:
     cdef ssize_t size = out.size
     cdef ndarray chunk
     cdef int generator = bit_generator.generator
-
+    # out is always c-contiguous, when out parameter is specified the checks
+    # ensure it
+    out = out.ravel()
```

Comment:
`out.ravel(order='A')` looks better, because f-contiguous `out` is allowed.

Reply:
good catch!